src/ir_Daikin.h

```diff
@@ -57,83 +57,87 @@
 #include "IRsend_test.h"
 #endif
 
-/*
-        Daikin AC map (i.e. DAIKIN, not the other variants)
-        byte 6=
-          b4:Comfort
-        byte 7= checksum of the first part (and last byte before a 29ms pause)
-        byte 13=Current time, mins past midnight, low bits
-        byte 14
-                b5-b3=Day of the week (SUN=1, MON=2, ..., SAT=7)
-                b2-b0=Current time, mins past midnight, high bits
-        byte 15= checksum of the second part (and last byte before a 29ms pause)
-        byte 21=mode
-                b7 = 0
-                b6+b5+b4 = Mode
-                        Modes: b6+b5+b4
-                        011 = Cool
-                        100 = Heat (temp 23)
-                        110 = FAN (temp not shown, but 25)
-                        000 = Fully Automatic (temp 25)
-                        010 = DRY (temp 0xc0 = 96 degrees c)
-                b3 = 1
-                b2 = OFF timer set
-                b1 = ON timer set
-                b0 = Air Conditioner ON
-        byte 22=temp*2   (Temp should be between 10 - 32)
-        byte 24=Fan
-                FAN control
-                b7+b6+b5+b4 = Fan speed
-                        Fan: b7+b6+b5+b4
-                        0×3 = 1 bar
-                        0×4 = 2 bar
-                        0×5 = 3 bar
-                        0×6 = 4 bar
-                        0×7 = 5 bar
-                        0xa = Auto
-                        0xb = Quite
-                b3+b2+b1+b0 = Swing control up/down
-                        Swing control up/down:
-                        0000 = Swing up/down off
-                        1111 = Swing up/down on
-        byte 25
-                        Swing control left/right:
-                        0000 = Swing left/right off
-                        1111 = Swing left/right on
-        byte 26=On timer mins past midnight, low bits
-        byte 27
-        b0-b3=On timer mins past midnight, high bits
-        b4-b7=Off timer mins past midnight, low bits
-        byte 28=Off timer mins past midnight, high bits
-        byte 29=Aux  -> Powerful (bit 1), Silent (bit 5)
-        byte 32=Aux2
-        b1: Sensor
-        b2: Econo mode
-        b7: Intelligent eye on
-        byte 33=Aux3
-        b1: Mold Proof
-        byte 34= checksum of the third part
-*/
+union DaikinESPProtocol{
```

Comment:
Can you please add similar comments to the `union` statements in this file as per: https://github.com/crankyoldgit/IRremoteESP8266/pull/1287 ?

Reply:
no problem